lib/airbrake/capistrano/tasks.rb

```diff
@@ -6,7 +6,8 @@
   namespace :airbrake do
     desc "Notify Airbrake of the deploy"
     task :deploy do
-      on roles(:all) do
+      role = roles(:all, select: :primary).first || roles(:all).first
```

Comment:
`.first` is needed in the event that `roles(:all, select: :primary)` returns an empty array, (no primary server was defined, which was your concern with the previous pull request).  The first element will be `nil`, which will then trigger the server to be pulled from the full listing (`rolles(:all)`).  I hope that makes sense.